Dynamo table update (#113)
Macrometacorp/docs: docs/collections/dynamo/create-dynamo-table.md

---
title: Create a Dynamo Table
sidebar_position: 10
---

> Dynamo Table collection is available for AWS Dynamo Database users. Macrometa’s platform can be used as a data store to acquire Global data distribution within milliseconds. To switch to GDN, the user need to change the connection URL, AccessKey, and SecretKey. AWS Dynamo DB SDK and CLI are compatible. Geo Distribution is set to Global by default.

Reviewer: ```suggestion
AWS Dynamo Database users can create Dynamo Table collections. You can use Macrometa as a datastore and achieve global data distribution within milliseconds. 

To switch to GDN, you must change the connection URL, `AccessKey`, and `SecretKey`. AWS Dynamo DB SDK and CLI are compatible. Geo-distribution is global by default.
```

> ## Create a Dynamo Table Collection  with the Console

Reviewer: ```suggestion
## Create a Dynamo Table Collection with the Console
```

> Follow these instructions to create a new collection using the GDN console web UI.

Reviewer: ```suggestion
To create a Dynamo Table collection:
```

> 1. Log in to your Macrometa account.

Reviewer: ```suggestion
1. Log into your Macrometa account.
```

Author: The original is actually correct. Weird, I know.

Reviewer: > The original is actually correct. Weird, I know.

I always thought it was more of a preference thing, but either way works for me.  Will remember this!
